tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -3,38 +3,46 @@
 Constrained Layout Guide
 ================================
 
-How to use constrained-layout to fit plots within your figure cleanly.
+Use Constrained layout to fit plots within your figure cleanly.
```

Comment:
My (poor) understanding of English is that "Constrained Layout" is a proper name and that we should capitalize both words. Can someone with more English competency please check this?

Reply:
I don't feel strongly about this and we could deal with the typography many ways.  However as we are encouraging layout='constrained' my attempt was to just emphasize "constrained".  Could instead emphasize _constrained_ layout or **constrained** layout.  The advantage of Constrained was that it works at the beginning of sentences ;-)

Reply:
Lowercase italics would also work.